packages/guides/src/Compiler/NodeTransformers/TocNodeWithDocumentEntryTransformer.php

```diff
@@ -96,42 +126,59 @@ public function getPriority(): int
         return 4500;
     }
 
-    private function isEqualAbsolutePath(DocumentEntryNode $documentEntry, string $file, string $currentPath, bool $glob): bool
+    public static function isEqualAbsolutePath(string $actualFile, string $expectedFile, string $currentFile, bool $glob): bool
```

Comment:
Please keep these methods private, as they are not used, and should not be used on other locations.